src/components/ProgressReport/PostProgressReport.tsx

```diff
@@ -19,8 +19,33 @@ const PostProgressReport = ({ className }: Props) => {
 	const { postData } = usePostDataContext();
 	const router = useRouter();
 
+	const getRelativeTime = (timestamp: { _seconds: number; _nanoseconds: number }): string => {
```

Comment:
Why this format? In backend convert firebase timestamp like this `created_at.toDate()`